addon/components/multi-faceted-search/facet.js

```diff
@@ -5,9 +5,11 @@ let Facet = Ember.Object.extend({
   type: "", // "checkbox" or "link"
   terms: [], // each term: { value: "", display: string.optional, count: num.optional, selected: boolean }
   emptyMessage: "Nothing to display",
+  hasError: false,
 
   checkbox: Ember.computed.equal('type', 'checkbox'),
   link: Ember.computed.equal('type', 'link'),
+  datetime: Ember.computed.equal('type', 'datetime'),
```

Comment:
Sorry, just trying to understand - when you select dates, do they appear in `applied-facets`?  And if so, how?  Our UX specs require all selected facets to be displayed in that annoying little bar ;)